dialogflow_v2beta1/proto/intent_pb2.py

```diff
@@ -2615,6 +3504,14 @@
             Synthesizes speech in Telephony Gateway.
         telephony_transfer_call:
             Transfers the call in Telephony Gateway.
+        rbm_text:
```

Comment:
Add rbm_text, rbm_standalone_rich_card, rbm_standalone_rich_card to Intent.Message (v2beta1)